package.json

```diff
@@ -9,7 +9,10 @@
   "author": "",
   "license": "ISC",
   "devDependencies": {
+    "@babel/core": "^7.9.0",
+    "@babel/preset-env": "^7.9.5",
     "gulp": "^4.0.0",
+    "gulp-babel": "^8.0.0",
     "gulp-sass": "^4.0.0"
```

Comment:
Why has this been changed?

Reply:
No idea. Uninstalled it for now, easy to reinstall if it turns out to have been required.